microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md

```diff
@@ -73,6 +73,23 @@ Issues are evaluated live while you edit your code.  As you type, you get feedba
 
 ![A possible issue being explained on an output element](microsoft-edge-devtools-extension-images/live-issue-reporting.png)
 
```

Comment:
I didn't see tagging errors.  To fix the doc build, I'd try merging "main" into the PR.

Reply:
Seems like the build worked fine this time.